install.sh

```diff
@@ -63,9 +63,25 @@ install_extension() {
         mkdir -p /tmp/extensions/resources
     fi
     cp -Rf resources/* /tmp/extensions/resources/
+
+    if [ -n "$ext_vars" ] && [ -n "$ext_name" ]; then
+        create_extension_js_file_with_vars
+    fi
+
     echo "UI extension installed successfully"
+
 }
 
+create_extension_js_file_with_vars() {
+  echo "Generating extension vars js file..."
+  ext_js_file_path="/tmp/extensions/resources/extension-$ext_name.js"
+  ext_js_file_name="vars-$(date +"%Y%m%d%H%M%S")"
+  js_file_path="${ext_js_file_path}/extension-${ext_js_file_name}.js"
+  js_code=$(echo "$ext_vars" | jq -r 'to_entries | map("\"" + .key + "\": \"" + .value + "\"") | join(", ")')
+  js_code="((window) => {\n  const vars = {\n    $js_code\n  };\n  window.ARGOCD_EXT_VARS = vars;\n})(window);"
+  echo "Exporting extension vars file at $js_file_path"
+  echo "$js_code" > "$js_file_path"
```

Comment:
This will fail if the target file doesn't exist

Reply:
added a check if directory  exist then create a  vars file or continue with extension installer with error msg
`
  if [ -d "$ext_installed_path" ]; then
      echo "Exporting extension vars file at $js_file_path"
      echo "$js_vars_wrap" > "$js_file_path"
  else
      echo "$ext_installed_path path doesn't exist, extension vars failed to be exported "
  fi
`